packages/braid-design-system/src/lib/components/Button/Button.tsx

```diff
@@ -306,11 +305,8 @@ export const ButtonText = ({
   iconPosition = 'leading',
   variant = 'solid',
   tone,
-  labelSpacing = true,
   bleed,
-}: ButtonProps & {
-  labelSpacing?: boolean;
```

Comment:
Do we want to call out the removal of this prop? Although I don't actually see it referenced anywhere in this repo.

Reply:
It is an internal component that we have refactored away from using and never cleaned up. So no comms needed.